source/_layouts/page-pattern.html

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+---
+<div id="main" class="col-md-12 clearfix" role="main">
+  <article class="post">
+    <header class="page-tabs-header">
+      <div class="row">
+        <div class="col-sm-12 col-md-6 cols-nopad-right">
+          <h1>{{ page.title }}</h1>
+        </div>
+        <div class="col-sm-12 col-md-6 cols-nopad-left cols-nopad-right">
+          {% if page.code == false %}
+            {% include nav-tabs-pattern.html codetab=false %}
+          {% elsif page.design == false %}
+            {% include nav-tabs-pattern.html designtab=false %}
+          {% else %}
+            {% include nav-tabs-pattern.html %}
+          {% endif %}
+        </div>
+      </div>
+    </header>
+    <section class="post_content">
+      <div class="tab-content">
+        <div role="tabpanel" class="tab-pane active" id="overview">
+          {% capture my_include %}{% include {{page.overview}} %}{% endcapture %}
+          {{ my_include | site_replace | markdownify }}
+        </div>
+        <div role="tabpanel" class="tab-pane" id="design">
+          {% capture my_include %}{% include {{page.design}} %}{% endcapture %}
+          {{ my_include | site_replace | markdownify }}
+        </div>
+        <div role="tabpanel" class="tab-pane" id="code">
+          {% include nav-tabs-code.html html=false %}
+          <div class="tab-content">
+            <div role="tabpanel" class="tab-pane nested active" id="angular">
+              <div ng-app="docsApp" ng-controller="DocsController" class="content">
+                <div ng-include src="'{{page.code}}'"></div>
+                <!-- % include {{page.code}} % -->
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </article>
+</div>
+<script>
+  var pageHeader = document.querySelector('.page-tabs-header h1'),
```

Comment:
My thought is that we continue to set the title as a variable in the template. Then, use a style to hide the first h1 each tab-pane selector. Otherwise, the JavaScript removes all h1 tags in the markup, not just the first title in each pane -- it's possible, although unlikely they would use multiple h1 tags.

For example, we could use this instead:
 
.tab-pane:not(.nested) > h1:first-child {
  display: none;
}